### DISCUSSION TITLE
How to change screenshot path ?

### DISCUSSION THREAD

**Discussion by sonichy** [linuxmint/discussions#9] [Questions] [answered]
Cinnamon press printscreen key save to $HOME/pictures, I want to save to $HOME/desktop, how to change ?

**Comment by claudiux** [comment]
First, take a screenshot and save it on your desktop.

Then, using 'dconf-editor' (`apt install dconf-editor`):

1. Go to /org/gnome/gnome-screenshot/
2. Copy the contents of the 'last-save-directory' field.
3. Paste it in the 'auto-save-directory' field.
4. Apply the changes.
5. Close dconf-editor.

That's right !

**Comment by clefebvre** [comment, marked as answer]
Open a terminal and run:

```
gsettings set org.gnome.gnome-screenshot auto-save-directory "file://$HOME/Desktop"
```

**Reply by clefebvre** [reply]
Essentially the same solution as the one from @claudiux but without installing anything and in only one step.

**Reply by claudiux** [reply]
Yes, and to know the registered value: `gsettings get org.gnome.gnome-screenshot auto-save-directory`

**Comment by claudiux** [comment]
@clefebvre 
I just made a PR to solve this problem, adding a Screenshots module in cinnamon-settings: https://github.com/linuxmint/cinnamon/pull/11656

**Reply by claudiux** [reply]
There are other settings to include in Screenshots module. Which should I add?

![Screenshot from 2023-05-07 20-39-54](https://user-images.githubusercontent.com/33965039/236696478-58d3ba30-8e86-4edc-96e0-6edb2dc97960.png)

**Reply by clefebvre** [reply]
Oh no, you jumped the gun on this. I don't think it's a good idea. I'm sorry you spent time implementing this.

gnome-screenshot is an upstream application.. I don't think it's up to Cinnamon to make it more easy to configure.. and if it was, it's arguable whether this needs to be accessible through UI, and not other settings.

Not to mention we're prototyping clicky, an Xapp screenshooter.

**Reply by claudiux** [reply]
Oh, I hadn't seen it: https://github.com/linuxmint/clicky
I'll try it.

You're right, it's not up to Cinnamon to manage external applications.

**Comment by sonichy** [comment]
Worked
`gsettings set org.gnome.gnome-screenshot auto-save-directory "/$HOME/Desktop"`
Send advice: https://gitlab.gnome.org/GNOME/gnome-screenshot/-/issues/231
How to disable flash animation ? It cause white mask on screenshot image.

**Comment by ineuw** [comment]
In saving the gnome screenshot, I thought that the _default location_ is restored automatically after a _Last place_ change? Is this possible? I ask because in my case, 98% of screenshots are to the default location.
